libs/blocks/utils/utils.js

```diff
@@ -26,4 +28,12 @@ const fetchWithTimeout = async (resource, options = {}) => {
   return response;
 };
 
-export { getLocalStorage, setLocalStorage, fetchWithTimeout };
+const updateLinkWithConfiguredLangRoot = (link) => {
```

Comment:
nit: name is a bit long, would `updateLinkWithLangRoot` work?

Reply:
`customLinkRoot`?

Reply:
I'll change it to `updateLinkWithLangRoot `

Reply:
addressed in https://github.com/adobecom/milo/pull/1101/commits/e2149877f76d990b95a22a56b442badd24f5c7b6